spec/models/service_template_ansible_playbook_spec.rb

```diff
@@ -0,0 +1,50 @@
+describe ServiceTemplateAnsiblePlaybook do
+  describe '#create_catalog_item' do
+    let(:user) { FactoryGirl.create(:user_with_group) }
+    let(:inventory_root_group) { FactoryGirl.create(:inventory_root_group) }
+    let(:ems) do
+      FactoryGirl.create(:automation_manager_ansible_tower, :inventory_root_groups => [inventory_root_group])
+    end
+    let(:config_script) { FactoryGirl.create(:configuration_script) }
+    let(:script_source) { FactoryGirl.create(:configuration_script_source, :manager => ems) }
+    let(:playbook) do
+      FactoryGirl.create(:configuration_script_payload,
+                         :configuration_script_source => script_source,
+                         :manager                     => ems,
+                         :inventory_root_group        => inventory_root_group,
+                         :type                        => 'ManageIQ::Providers::AnsibleTower::AutomationManager::Playbook')
+    end
+    let(:service_template_catalog) { FactoryGirl.create(:service_template_catalog) }
+    let(:catalog_item_options) do
+      {
+        :name                     => 'test_ansible_catalog_item',
+        :description              => 'test ansible',
+        :service_template_catalog => service_template_catalog,
+        :config_info              => {
+          :provision         => {
+            :new_dialog_name => 'test_dialog',
+            :hosts           => 'many',
+            :credential      => 6,
```

Comment:
to match our documentation `:credential_id => 6` although it is not used for testing